Jenkinsfile

```diff
@@ -1,6 +1,10 @@
 if (infra.isRunningOnJenkinsInfra()) {
     // ci.jenkins.io
-    buildPlugin(platforms: ['linux'])
+    buildPlugin(configurations: [
+    [ platform: "linux", jdk: "8", jenkins: null ],
+    [ platform: "linux", jdk: "8",  jenkins: "2.164.1", javaLevel: "8" ],
+    [ platform: "linux", jdk: "11", jenkins: "2.164.1", javaLevel: "8" ],
```

Comment:
What is the deal with `javaLevel`? The POM already specifies

```xml
<java.level>8</java.level>
```

so why would we need to override that?

Reply:
https://github.com/jenkins-infra/pipeline-library/pull/86#discussion_r269581744 FTR